main.py

```diff
@@ -684,10 +684,9 @@ def handle_flow_mod_error(self, event):
         command = event.content.get('error_command')
         if command != 'add':
             return
-        evc_id = f'{flow.cookie:x}'
-        evc = self.circuits.get(evc_id)
+        evc = self.circuits.get(EVC.get_id_from_cookie(flow.cookie))
         if evc:
-            evc.remove_current_flows()
```

Comment:
@viniarck I don't think we should use `evc.remove()` instead of `evc.remove_current_flows()` because this way the evc will be disabled. I think the EVC should be deactivated instead of disabled (which is already done by `remove_current_flows()`. What do you think @viniarck and @ajoaoff ?

Reply:
Right. Yeah, the `remove_current_flows()` should suffice, initially I switched it to the `remove()` to also `disable` it ir order not to allow a redeploy since ofpt errors should be deterministic, so redeploys would result in the same error if operators wree to redeploy, but as long as operators use this carefully it shouldn't be an issue, and in fact, if one day their switch vendors fix the ofpt error or if they replace their switch, they'd have to reploy anyway. I agree let's maintain the  `remove_current_flows()`, I'll push a commit reverting this.

Reply:
@italovalcy https://github.com/kytos-ng/mef_eline/pull/106/commits/9ebd4704991ad4a8cd2461b33e2305a42720ed75

Reply:
Great! Thanks @viniarck !! Just another aspect on why not disable the EVC is: it can be using dynamic paths, and eventually pathfinder returns a not path not including the problematic switch which returned the oft error. However, I got your worry about keep trying to redeploy an EVC which always returns failure. We actually have a situation like this on our current SDN orchestrator and we have to rely on the logs processing to identify and fix such situations.

Reply:
Just registered an issue to handle this in the future: #107

Reply:
Got it. Also, thanks for mapping htis @italovalcy